playbarComponent/client/src/components/App.jsx

```diff
@@ -51,12 +52,8 @@ class App extends Component {
   //initialize array of songs
   loadSongs() {
     let list = [];
-    axios
-
-      .get(
-        'http://ec2-18-188-157-241.us-east-2.compute.amazonaws.com:5002/query/getSong/song_0001'
-        // 'http://localhost:5002/query/getSong/song_0001'
-      )
+    axios.get`http://ec2-18-189-170-126.us-east-2.compute.amazonaws.com:5002/query/getSong/${this.props.song_id}`()
```

Comment:
Nice work implementing the EC2 instances. If you get the chance, try implementing elastic IPs so that the IPV4s wont change